pynecone/components/disclosure/accordion.py

```diff
@@ -28,13 +28,23 @@ class Accordion(ChakraComponent):
     reduce_motion: Var[bool]
 
     @classmethod
-    def create(cls, *children, items=None, icon_pos="right", **props) -> Component:
+    def create(
+        cls,
+        *children,
+        items=None,
+        icon_pos="right",
+        allow_multiple: Var[bool],
```

Comment:
Can we add type annotations for all the props. The `allow_multiple` and `allow_toggle` should be Optional